templates/admin/src/lib/react-hook-toggle/ReactHookToggle.ts

```diff
@@ -0,0 +1,15 @@
+import { useCallback, useState } from 'react';
+
+/**
+ * Hook used to help handle toggle actions.
+ */
+export default function useToggle(initialState: boolean = false): [boolean, () => void] {
+  const [value, setValue] = useState(initialState);
+
+  return [
+    value,
+    useCallback(() => {
+      setValue(!value);
+    }, [value]),
```

Comment:
```suggestion
    }, []),
```

comme ça ton memo n'est pas màj à chaque changement de valeur !